gem2rpm/smart_proxy_plugin.spec.erb

```diff
@@ -1,17 +1,18 @@
-# FIXME:
-#   1. Edit foreman-proxy requirement and specify minimum version
-#   2. Delete these lines
-#
+# Generated from <%= package.spec.file_name %> by gem2rpm -*- rpm-spec -*-
```

Comment:
We actually dropped these in 8d6a5d23c90c69c796e359571545dead05822644 so why are you re-introducing them?